src/utils/AccountUtils.ts

```diff
@@ -0,0 +1,97 @@
+/*-
+ *
+ * Hedera Local Node
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import {
+  AccountCreateTransaction,
+  AccountId,
+  AccountInfo,
+  AccountInfoQuery,
+  Client,
+  Hbar,
+  PrivateKey,
+  PublicKey,
+  TransferTransaction
+} from '@hashgraph/sdk';
+import { IAccountProps } from '../configuration/types/IAccountProps';
+import { getPrivateKey, KeyType } from '../configuration/types/IPrivateKey';
+
+/**
+ * Provides utility methods for working with accounts.
+ */
+export class AccountUtils {
+
+  public static async createAccountFromProps(account: IAccountProps,
+                                             client: Client): Promise<{privateKey: PrivateKey, accountInfo: AccountInfo}> {
+    const keyType = account.privateKey ? account.privateKey.type : KeyType.ECDSA;
+    const privateKey = account.privateKey ? getPrivateKey(account.privateKey) : PrivateKey.generateECDSA();
+
+    let accountInfo: AccountInfo;
+    if (keyType === KeyType.ED25519) {
+      accountInfo = await this.createAccount(privateKey.publicKey, account.balance, client);
+    } else {
+      const accountId = privateKey.publicKey.toAccountId(0, 0);
+      accountInfo = await this.createAliasedAccount(accountId, account.balance, client);
+    }
+
+    return { accountInfo, privateKey };
+  }
+
+  /**
+   * Creates an account with the given properties.
+   * @param aliasAccountId The alias ID of the account to create.
+   * @param initialBalance The initial balance of the account.
+   * @param client The client to use for creating the account.
+   */
+  public static async createAliasedAccount(aliasAccountId: AccountId,
```

Comment:
In the accountCreationState we have methods for creating accounts. May you check if the ones you created can replace the ones there. Just so we do not have the same functiionality over and over again